CMakeLists.txt

```diff
@@ -293,7 +302,7 @@ ENDIF()
 
 # MiniTensor
 LIST(FIND Trilinos_PACKAGE_LIST MiniTensor MiniTensor_List_ID)
```

Comment:
I think we no longer need MiniTensor. We should probably remove this

Reply:
Agreed. I just removed this.

Reply:
This is gone.